src/utils/dash.js

```diff
@@ -0,0 +1,63 @@
+import root from './root';
```

Comment:
I'd like to rename the file to `underscore.js` or `lodash.js` to at least reference some level of functionality where these methods are inherited. `dash` could one day be some type of style or object element? Or maybe some video encoding type that is used as a texture -- I'm highly speculating, but I really like the dash term as a top-level-accessible-property for a library 😅

Reply:
I'll rename it to `underdash`.